### PR TITLE
CLN: Remove unused fixtures

### PR DESCRIPTION
**Pull request by dsaxton**

### CODE REVIEW

File: pandas/tests/window/conftest.py [hunk 1]
```diff
@@ -42,21 +42,11 @@ def arithmetic_win_operators(request):
     return request.param
 
 
-@pytest.fixture(params=["right", "left", "both", "neither"])
-def closed(request):
```
Comment (jreback) [inline pandas/tests/window/conftest.py:46]: this a duplicate yes?

Comment (jreback) [reply pandas/tests/window/conftest.py:46]: as this is generally a fixture we do want

File: pandas/tests/window/conftest.py [hunk 2]
```diff
@@ -42,21 +42,11 @@ def arithmetic_win_operators(request):
...
-    return request.param
-
-
 @pytest.fixture(params=[True, False])
 def center(request):
     return request.param
 
 
-@pytest.fixture(params=[None, 1])
-def min_periods(request):
```
Comment (jreback) [inline pandas/tests/window/conftest.py:56]: why are u removing this one?

File: pandas/tests/window/conftest.py [hunk 3]
```diff
@@ -75,14 +65,6 @@ def nopython(request):
     return request.param
 
 
-@pytest.fixture(
-    params=[pytest.param("numba", marks=td.skip_if_no("numba", "0.46.0")), "cython"]
-)
```
Comment (jreback) [inline pandas/tests/window/conftest.py:80]: this is also needed

**Review by jreback — CHANGES_REQUESTED** [review]
prefer to be less aggressive on these removals 

most are ok - noted some

File: pandas/tests/window/conftest.py (hunk 2 above)
Comment (dsaxton) [reply pandas/tests/window/conftest.py:56]: No special reason, was only removing if unused. Adding back these window fixtures.

**Comment by jbrockmendel** [comment]
LGTM

**Comment by WillAyd** [comment]
Does pytest expose a way to figure out what is / isn't being used?

**Comment by dsaxton** [comment]
> Does pytest expose a way to figure out what is / isn't being used?

There's a plugin (which is what I was using here) but not sure how well-maintained it is: https://github.com/jllorencetti/pytest-deadfixtures

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

also as a possible issue / followup. we ideally want to surface the same / similar fixtures to the top level (and similarly push down ones that are ownly used beneath a particular sub-tree to a lower level). not sure if there is an easy way to do this.

**Pull request merged**